docker/bitcoin/electrs/Dockerfile

```diff
@@ -1,29 +1,22 @@
-FROM rust:alpine3.13 AS electrsbuild
+FROM rust:1.69-alpine3.17 AS electrsbuild
 
 RUN apk add --no-cache clang-dev build-base linux-headers git
 RUN rustup component add rustfmt
-
 WORKDIR /workdir
-
 RUN git clone --no-checkout https://github.com/Blockstream/electrs.git
-
 WORKDIR electrs
-
-RUN git checkout a33e97e1a1fc63fa9c20a116bb92579bbf43b254
-
+RUN git checkout 20e42862a57f0d45ce48a9956a361e7193b9e97d
```

Comment:
What is specific about this commit? Is it the most recent tag version or some "last working" version? Can we add a comment? Same question about other `Dockerfile`s.